src/functionalTest/java/uk/gov/hmcts/dts/fact/SearchEndpointTest.java

```diff
@@ -85,7 +92,7 @@ public void nameSearchShouldSupportWelsh() {
     @Test
     public void shouldRetrieve10CourtReferenceByPostcodeAndServiceAreaSortedByDistance() {
         final String serviceArea = "claims-against-employers";
-        final var response = doGetRequest(SEARCH_ENDPOINT + "results?postcode=OX1 1RZ&serviceArea=" + serviceArea);
+        final var response = doGetRequest(SEARCH_ENDPOINT + "results?includeClosed=true&postcode=OX1 1RZ&serviceArea=" + serviceArea);
```

Comment:
We probably want to keep this as it was by default right? as default is true? Either way its probably fine